### ISSUE TITLE
pip install incompleted

### ISSUE THREAD

**Issue by markuman** [uktrade/mbtiles-s3-server#1]
After trying to start ` python3 -m mbtiles_s3_server`

I got 

`FileNotFoundError: [Errno 2] No such file or directory: '/usr/local/lib/python3.10/dist-packages/mbtiles_s3_server/vendor/osm-bright-gl-style@1.0.0/style.json'` 

`osm-bright-gl-style` is missing

```
ubuntu@osm:~$ sudo find /usr/local/lib/python3.10/ -name style.json
/usr/local/lib/python3.10/dist-packages/mbtiles_s3_server/vendor/maptiler-toner-gl-style@1.0.0/style.json
/usr/local/lib/python3.10/dist-packages/mbtiles_s3_server/vendor/positron-gl-style@1.0.0/style.json
/usr/local/lib/python3.10/dist-packages/mbtiles_s3_server/vendor/osm-liberty@1.0.0/style.json
/usr/local/lib/python3.10/dist-packages/mbtiles_s3_server/vendor/fiord-color-gl-style@1.0.0/style.json
/usr/local/lib/python3.10/dist-packages/mbtiles_s3_server/vendor/maptiler-3d-gl-style@1.0.0/style.json
/usr/local/lib/python3.10/dist-packages/mbtiles_s3_server/vendor/dark-matter-gl-style@1.0.0/style.json
/usr/local/lib/python3.10/dist-packages/mbtiles_s3_server/vendor/maptiler-terrain-gl-style@1.0.0/style.json
/usr/local/lib/python3.10/dist-packages/mbtiles_s3_server/vendor/maptiler-basic-gl-style@1.0.0/style.json

```

using python 3.10 on ubuntu 22.04

**Comment by michalc** [comment]
Can reproduce the issue - investigating

**Comment by michalc** [comment]
Fixed in [v0.0.12](https://pypi.org/project/mbtiles-s3-server/) (Feel free to re-open if it's still an issue)
